src/main/java/frc/robot/note_tracking/NoteTrackingManager.java

```diff
@@ -355,18 +355,42 @@ public boolean mapContainsNote() {
   }
 
   public void addNoteToMap(Translation2d pose) {
-    noteMap.add(new NoteMapElement(Timer.getFPGATimestamp() + NOTE_MAP_LIFETIME, pose));
+    noteMap.add(new NoteMapElement(Timer.getFPGATimestamp() + NOTE_MAP_LIFETIME, pose, 10));
+  }
+
+  private NoteMapElement updatedHealth(NoteMapElement oldElement) {
+    return new NoteMapElement(
+        oldElement.expiresAt(), oldElement.noteTranslation(), oldElement.health() - 1);
```

Comment:
i don't think this method is really doing anything helpful, i would just remove it